dev-guide.md

```diff
@@ -0,0 +1,165 @@
+# K8ssandra Developer Guide
+
+Ready to write some code?  Jump straight to the one of our other guides:
+
+* [K8ssandra Chart Development Quick Start](dev-quick-start.md)
+* [Contributing to Cass Operator](https://github.com/k8ssandra/cass-operator/blob/master/README.md#contributing)
+
+## Overview
+
+K8ssandra is a series of projects orchestrated to deliver a cloud-native Apache Cassandra deployment on Kubernetes.  To learn more about the project in general jump over to the project's main [README](README.md).
+
+This guide is aimed at introducing developers to the various technologies and sub-projects involved in K8ssanrdra as well as some of the processes that the project uses to plan and prioritize our work.
+
+## Learning Resources
+
+To develop within the K8ssandra ecosystem there are some high-level topics that you'll want to gain experience with:
+
+* Kubernetes
+* Kind
+* Helm
+* Kubernetes Operator Development
+* Go
+* Apache Cassandra
+* Medusa for Apache Cassandra
+* Reaper for Apache Cassandra
+* Git/GitHub
+
+There are a lot of great learning resources out there related to all of these topics.  Here is a small collection of resources that our team has found useful over time.
+
+(If you have any you'd like to add here, we'd love to have the contribution!)
+
+* [Kubernetes Documentation](https://kubernetes.io/docs/home/)
+* [Kind Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start/)
+* [Helm Documentation](https://helm.sh/docs/)
+* [Kubebuilder Book](https://book.kubebuilder.io/)
+* [Building Operators with Go](https://sdk.operatorframework.io/docs/building-operators/golang/)
+* [Tutorial: Get Started with Go](https://golang.org/doc/tutorial/getting-started)
+* [DataStax Academy](https://www.datastax.com/dev/academy)
```

Comment:
I would recommend this specific course in addition to (or instead of) Academy, whichever you prefer https://www.datastax.com/learn/apache-cassandra-operations-in-kubernetes

Reply:
Thanks, I've swapped that one in.